lib/mapObjects/CGHeroInstance.cpp

```diff
@@ -62,52 +62,44 @@ ui32 CGHeroInstance::getTileCost(const TerrainTile &dest, const TerrainTile &fro
 	unsigned ret = 100;
```

Comment:
There is configuration of move cost based on terrain type. Should it be used here?